### PR TITLE
features of history like and watchlater added

### PR DESCRIPTION
**Pull request by ShreyashMahajan**
Featues of the following pages are added.
1. Liked page
2. Watchlater page
3. history page

### CODE REVIEW

**Review by paarthbajaj — CHANGES_REQUESTED** [review]
Code is well structured and saw good practices. Few suggestions: no need to use justify-center in  videoCard-container div. Also you need to make corrections in your code, because when I click on the three dots to check the functionality, your app is crashing. Basically, I am unable to test your features. One more thing, I am unable to login or signup as it is necessary to view features like History,etc.

**Comment by ShreyashMahajan** [comment]
> Code is well structured and saw good practices. Few suggestions: no need to use justify-center in videoCard-container div. Also you need to make corrections in your code, because when I click on the three dots to check the functionality, your app is crashing. Basically, I am unable to test your features. One more thing, I am unable to login or signup as it is necessary to view features like History,etc.

sorry for that,  i added .env file in my project, but it was not coming here, Now i fixed it. You can test features now.

**Review by paarthbajaj — APPROVED** [review]
The app is working now. Appreciate the quick fix. Great work. Well done.

**Comment by ShreyashMahajan** [comment]
> The app is working now. Appreciate the quick fix. Great work. Well done.

thank you

File: src/App.js [hunk 1]
```diff
@@ -6,16 +6,43 @@ import { History } from "./pages/history/history";
 import { Home } from "./pages/home/home";
 import { Playlist } from "./pages/playlist/playlist";
 import { WatchLater } from "./pages/watchLater/watchLater";
+import { Routes, Route } from 'react-router-dom';
+import { LikedPage } from "./pages/likedVideo/likedVideo";
+import { RequireAuth } from "./components/requireAuth/requireAuth";
+import { ToastContainer, toast } from 'react-toastify';
+import 'react-toastify/dist/ReactToastify.css';
```
Comment (SaurabhChawda) [inline src/App.js:13]: Create index file and Exports files from there.

**Review by SaurabhChawda — APPROVED** [review]
Design and code looks fine to me. Just Create Index File for export files.Liked that you have Created service file for Api calls.You can merge it.

File: src/App.js (hunk 1 above)
Comment (ShreyashMahajan) [reply src/App.js:13]: ok, i will do that, thank you

**Pull request merged**